server/router.go

```diff
@@ -58,6 +59,16 @@ func NewServer(params ServerParams) *fiber.App {
 	app.Use(recover.New())
 	params.Logger.Info("Added Recovery Middleware.")
 
+
+	// Middleware for healthcheck
+	app.Use(healthcheck.New(healthcheck.Config{
+		LivenessProbe: func(c *fiber.Ctx) bool {
+			return true
+		},
+		LivenessEndpoint: "/health",
```

Comment:
Please change to livez, as this is a liveness endpoint
See: [Kubernetes API Health Endpoints Standard](https://kubernetes.io/docs/reference/using-api/health-checks/)